miso-web/src/main/webapp/scripts/form_sample.js

```diff
@@ -359,7 +359,7 @@ FormTarget.sample = (function($) {
             }]
           }, {
             title: 'Slide',
-            include: config.detailedSample && object.sampleClassAlias === 'Slide',
+            include: config.detailedSample && object.sampleSubcategory === 'Slide',
```

Comment:
Need to add the `sampleSubcategory` field to `DetailedSampleDto`. Currently, all of the subcategory fields are missing from the Edit Sample page

Reply:
Okay. I think this is fixed.